doc/020_installation.rst

```diff
@@ -109,6 +109,15 @@ On FreeBSD (11 and probably later versions), you can install restic using ``pkg
 
     # pkg install restic
 
+openSUSE
+========
+
+restic can be installed using ``zypper`` package manager:
```

Comment:
Just a couple of small things, can you:

1. Start the sentence like the others above and below, i.e. "On ...", and in there mention from which version(s) of OpenSUSE this installation can be done (as you mentioned it in the issue)?
2. Add "the" after "installed using".

Once done, please amend/rebase/squash the commits in this branch into one (let us know if you need help with that). Thanks a lot!

Reply:
@rawtaz Update openSUSE version(s) and the sentence :)